### ISSUE TITLE
Create a cacheing *frontend* ontology lookup service

### ISSUE THREAD

**Issue by andrewpatto** [umccr/elsa-data#50]
We are using proper coding systems for things like gene pickers and disease codes. This will use a service like "ontoserver" direct from the front end code.

There is an example of its use from the Disease Chooser react component (where it looks up mondo diseases).

I cobbled together a horrible async/cache version in a hurry to get a demo working - but it needs to be done better.

I am thinking a proper cache (rather than a dictionary as it currently is)

https://github.com/isaacs/node-lru-cache

(or other??)

And the ability to pre-populate with static sets (ala country codes and DUO codes - which are codesets that we probably won't ever have in ontoserver - but we should access via the same pattern)

See `helpers/ontology-helper.ts` for the current working implementation.

Basically this task is to tidy that code up into a frontend side ontology lookup service..

**Comment by DoxasticFox** [comment]
Resolved by https://github.com/umccr/elsa-data/pull/72.
